std/haxe/web/HTTPMethod.hx

```diff
@@ -0,0 +1,89 @@
+package haxe.web;
+/**
+* HTTP defines methods (sometimes referred to as verbs) to indicate the desired action to be performed on the identified resource. What this resource represents, whether pre-existing data or data that is generated dynamically, depends on the implementation of the server. Often, the resource corresponds to a file or the output of an executable residing on the server. The HTTP/1.0 specification[11] defined the GET, POST and HEAD methods and the HTTP/1.1 specification[12] added 5 new methods: OPTIONS, PUT, DELETE, TRACE and CONNECT. By being specified in these documents their semantics are well known and can be depended upon. Any client can use any method and the server can be configured to support any combination of methods. If a method is unknown to an intermediate it will be treated as an unsafe and non-idempotent method. There is no limit to the number of methods that can be defined and this allows for future methods to be specified without breaking existing infrastructure.
+* @class HTTPMethod
+*/
+@:enum abstract HTTPMethod(String) from String to String  {
+
+/**
+ * The POST method requests that the server accept the entity enclosed in the request as a new subordinate of the web resource identified by the URI. The data POSTed might be, for example, an annotation for existing resources; a message for a bulletin board, newsgroup, mailing list, or comment thread; a block of data that is the result of submitting a web form to a data-handling process; or an item to add to a database
+ * @property POST
```

Comment:
FWIW, I'm not a big fan of this extra stuff in the doc. 

I agree there's value in mentioning what POST means. But using four lines to explain for each var on an `@:enum abstract` is static, readonly, of the type of the abstract and is a property of the same name as the variable is just noise. For `MIME` it makes the diff so big that GitHub doesn't even render it by default. My advice is to keep the documentation as short as possible without losing information. It should *expand* on what the code says already. I think there'd be more value in adding comments to `HTTPStatus` than having this kind of annotation all over the place.